cas/audit-incident-report.md

```diff
@@ -0,0 +1,88 @@
+
+When audits are performed, an audit statement may document qualifications or non-conformities (i.e., findings) that were identified during the audit. In some cases, each finding disclosed in the audit statement may have been self-reported by a CA Owner or third party in a previous [Incident Report](incident-report). In cases where **each** finding was *not* previously recorded in an Incident Report, CA Owners MUST create a new Bugzilla issue by filling out the "Summary" and "Description" fields of [this form](https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&product=CA%20Program&component=CA%20Certificate%20Compliance&bug_type=task). The Summary field MUST include the CA Owner name, a colon, and "Findings in 20XX Audit", where XX is the year the audit period or point-in-time ended (e.g., "CA ABC: Findings in 2024 Audit").
+
+To create the audit incident report, copy the Markdown template below and fill out each section according to the following instructions and requirements.
+
+If the audit includes multiple findings, the audit incident report MUST include a separate Finding, Root Cause Analysis, and Action Items Sections for each finding. 
+
+### Audit Incident Report Template
+
+```markdown
+
+## Audit Incident Report
+
+### Finding # (i.e., #1)
+<!---
+
+**Expectations:**
+
+- This section MUST contain a description of the nature of each finding. This provides context for the reader to understand the nature of the non-compliance identified by the Auditor.
+
+-->
+
+#### Root Cause Analysis
+<!---
+
+**Expectations:**
+
+- A root cause is defined as a factor that caused a nonconformance and should be permanently eliminated through process improvement.
+- This section MUST contain a detailed analysis of the conditions which combined to give rise to the issue.
+- It is unusual for an incident to have a single root cause; often there is a confluence of several issues such as a software bug, insufficient checks, and a malformed request.
+- Make sure that all contributing causes are identified and described, including noting when they first arose and how they avoided detection until they were discovered or identified. 
+
+##### Issue [#]: [Title]
+- **Description:** [A detailed description of the specific issue.]
+- **Issue Onset:** [Date when the issue began.]
+- **Issue Detection:** [Date when the issue was detected or otherwise made known.]
+- **Issue Resolution:** [Actual or planned date when the issue will be considered resolved.]
+- **Symptoms that Led to Detection**: [A detailed description of the circumstances that led to the detection of the issue.]
+- **How the Issue Avoided Detection**: [A detailed description of how the issue was not detected earlier.]
+- **Root Cause Analysis Methodology Used**([OPTIONAL, but RECOMMENDED]): [A description of the methodology used to derive the issue described above (e.g., "5-Whys", Fishbone Diagram, Pareto Analysis, etc.)]
+
+-->
+
+
+#### Action Items
+<!---
+
+**Expectations:**
```

Comment:
One challenge that we've had with Action Items historically is that there has been no way to tell if they were effective in actually changing anything. An example of this is a Subscriber education effort around revocation, or process updates around misissuance: when proposed as Action Item in response to an Incident, there's really no way for the community to know if the AI was effective against the root causes (such as Subscriber ignorance or insufficient process).  I would propose that each Action Item include evaluation criteria: how will the CA measure (perhaps qualitatively, such as via surveys, or operationally via simulations and "drill" scenarios) the impact of the Action Item on the associated Root Cause? In my ideal world, closing out the incident would require reporting on the measured effectiveness, and committing to reporting on the same measurement after 6 and 12 months to ensure that the changes stuck. (The "stickiness" of changes is of additional concern when the AI is about education or "cultural" change, IMO.)
This would not only aid with community oversight, but also make it easier for other CAs to learn from each other's remediation efforts: if we don't know if a thing worked, we don't know if other CAs should adopt the thing, either prophylactically or in response to a similar incident in the future. 

The brevity of the sample AIs probably works against our interests here, in that they seem to ask for just labels of activity rather than clear statements of the desired state to be reached. I think that as a community we care more about what the result will be than what the activity will be to achieve it.

Reply:
We attempted to address this using a new Action Item field in #187, described below:

> Evaluation criteria: Describe how the CA Owner will measure the effectiveness of the Action Item in addressing the Root Cause. Include how the public can also measure this impact, if applicable. CA Owners SHOULD prioritize objective and publicly measurable evidence (i.e., evidence that can be independently verified by the public, such as through Certificate Transparency log data, audit statements, CA policy documents, and public communications (e.g., website content, surveys, etc.)). While external metrics are preferred, these Guidelines recognize that some Action Items may have limited measurable outcomes. When objective metrics are not readily available, CA Owners SHOULD provide a qualitative assessment of the Action Item's effectiveness and explain how they will monitor its impact. Even after an Incident Report has been closed, CA Owners are strongly encouraged to provide periodic updates (e.g., 3, 6, and 12 months post-closure, or at other determined appropriate intervals) related to the ongoing efficacy of the Action Item, as measured against the evaluation criteria. This helps demonstrate a continued commitment to improvement and transparency.